### PR TITLE
Feature/new sim

### PR DESCRIPTION
**Pull request by raghavauppuluri13**

### CODE REVIEW

**Comment by Isaac-F24** [comment]
New arenas for sim:

Changes the launch file for simulation to support a boolean arguemnt labeled `KSC_arena`.
The default is true, and it represents whether or not to use an arena resembling the Kennedy Space Center's field or one resembling UCF's arena.
In addition, changes the initial starting value of the robot (to the bottom left).

Changes in submodule `mining_arena_gazebo`: See [submodule commit](https://github.com/PurdueLunabotics/mining_arena_gazebo/commit/b758d83613450e51c5b06bc1e5609781917babce).

Separates the arena model into two separate folders: NASA (KSC) and UCF. See either guidebook for arena information.
The dimensions of each model matches the arena specifications. (Both are quite a bit larger)

Also changes each model's `.world` file which controls obstacle placement.

File: lunabot_bringup/launch/sim.launch [hunk 1]
```diff
@@ -1,7 +1,15 @@
 <?xml version="1.0"?>
 <launch>
-  <arg name="world_name" default="$(find mining_arena_gazebo)/worlds/arena.world"/>
-  <arg name="initial_pose" default="-x 1.0 -y 0 -z 0"/>
+
+  <!-- Set the arena to either KSC or UCF arena based on the boolean argument. -->
+  <arg name="KSC_arena" default="true"/>
```
Comment (raghavauppuluri13) [inline lunabot_bringup/launch/sim.launch:5]: nit: args and params are usually all undercase (snake case)

**Comment by raghavauppuluri13** [comment]
Just for reference:
KSC pit: 
image
UCF pit: 
image

**Comment by raghavauppuluri13** [comment]
@Isaac-F24 just a quick comment. after that, looks good to merge.

**Pull request merged**